common/src/pool_pair.rs

```diff
@@ -331,4 +411,26 @@ where
     fn program_id(&self) -> Pubkey {
         stakedex_interface::ID
     }
+
+    fn get_accounts_len(&self) -> usize {
+        // Pick a single direction
+        1 + <dyn WithdrawStake>::accounts_len(&self.p1) + DepositStake::accounts_len(&self.p2) + 1
```

Comment:
should this be one sided only? instead of combining two? like Math.max(withdraw,deposit)

Reply:
It withdraw the stake then deposit it elsewhere, so all accounts will be present.